lib/permanent_records.rb

```diff
@@ -51,9 +50,13 @@ def destroy(force = nil)
 
     private
 
+    def get_deleted_record
+      self.class.unscoped.find(id)
+    end
```

Comment:
Use this to avoid using `stub_chain` of `Rspec` consider like code smell see http://www.relishapp.com/rspec/rspec-mocks/v/3-4/docs/working-with-legacy-code/message-chains